tests/common/utils.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains some utils for testing purposes"""
+
+import time
+from contextlib import contextmanager
+
+DEFAULT_SLEEP = 0.0001
+DEFAULT_TIMEOUT = 3
+
+
+class TimeItResult:
+    """ class to store execution time for timeit_context """
+
+    def __init__(self):
+        self.time_passed = -1
+
+
+@contextmanager
+def timeit_context():
+    """
+        context manager to measure execution time of code in context
+
+        :return TimeItResult
+
+        example:
+        with timeit_context() as result:
+            do_long_code()
+        print("Long code takes ", result.time_passed)
+    """
+
+    result = TimeItResult()
+    started_time = time.time()
+    try:
+        yield result
+    finally:
+        result.time_passed = time.time() - started_time
+
+
+class AgentTool:
+    def __init__(self, agent):
```

Comment:
Is this an agent or AEA? Please add function signature (types) and also rename class to `AEATool` if the `AEA` is used.